ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -176,7 +172,12 @@ Item {
 
         onCountChanged: d.markAllMessagesReadIfMostRecentMessageIsInViewport()
 
-        onVisibleChanged: d.markAllMessagesReadIfMostRecentMessageIsInViewport()
+        onVisibleChanged: {
+            if (visible) {
+                messageStore.addNewMessagesMarker()
+            }
```

Comment:
This seems redundant as it is already done on the nim side:
https://github.com/status-im/status-desktop/blob/6e188de21af5cdfa96625ad37d5fafd61b91859d/src/app/modules/main/chat_section/chat_content/module.nim#L392-L395

Reply:
fixed